### PR TITLE
Initial changes to jwt version

### PR DESCRIPTION
**Pull request by jarrodwatts**

### CODE REVIEW

File: pages/index.jsx [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+        !
+      </p>
+
+      {address ? (
+        <button
+          className={styles.mainButton}
+          style={{ width: "fit-content", paddingRight: 16, paddingLeft: 16 }}
+          onClick={signIn}
+        >
+          Sign In With Ethereum
```
Comment (adam-maj) [inline pages/index.jsx:66]: We might even want to call this something more agnostic like `Login with Wallet` - although up to you, b/c ofc people do know about Sign in with Ethereum.

File: pages/restricted-page.jsx [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+  // Get addresses' balance of token ID 0
+  const balance = await editionDrop.balanceOf(address, 0);
+
+  // If the balance is greater than 0, then the user can see the content
+  const canSeeContent = balance > 0;
+
+  let content;
+  if (canSeeContent) {
+    // Here is where you would make a request to load some content, such as from a database.
+    content = "Here is some restricted content since you own an NFT!";
```
Comment (adam-maj) [inline pages/restricted-page.jsx:80]: Would be cool to do an image here to get the point across more

File: pages/api/authenticate.js [hunk 3]
```diff
@@ -0,0 +1,38 @@
+import { ThirdwebSDK } from "@thirdweb-dev/sdk";
+
+const authenticate = async (req, res) => {
```
Comment (adam-maj) [inline pages/api/authenticate.js:3]: This is the endpoint you could hit on page load/whenever you want to check if user is still logged in. Basically if it doesn't return `ok` HTTP status then user isn't logged in.

Comment (jarrodwatts) [reply pages/api/authenticate.js:3]: included this in latest commit

File: pages/index.jsx (hunk 1 above)
Comment (jarrodwatts) [reply pages/index.jsx]: done

File: pages/restricted-page.jsx (hunk 2 above)
Comment (jarrodwatts) [reply pages/restricted-page.jsx:80]: where would you get the image from?

Comment (adam-maj) [reply pages/restricted-page.jsx:80]: good point

**Pull request merged**